edx-platform/pearson-bsg-theme/lms/templates/header/user_dropdown.html

```diff
@@ -19,6 +19,10 @@
 self.real_user = getattr(user, 'real_user', user)
 profile_image_url = get_profile_image_urls_for_user(self.real_user)['medium']
 username = self.real_user.username
+
+if configuration_helpers.get_value('SHOW_PROFILE_NAME_INSTEAD_OF_USERNAME', False):
```

Comment:
Why don't you also validate `if getattr(self.real_user, 'profile', None)` here.? @Squirrel18 

```suggestion
if getattr(self.real_user, 'profile', None) and configuration_helpers.get_value('SHOW_PROFILE_NAME_INSTEAD_OF_USERNAME', False):
```

Reply:
Not necessary. @Jacatove